ci/build/build-release.sh

```diff
@@ -98,10 +98,6 @@ EOF
   # yarn to fetch node_modules if necessary without build scripts running.
   # We cannot use --no-scripts because we still want dependent package scripts to run.
   jq 'del(.scripts)' < "$VSCODE_SRC_PATH/package.json" > "$VSCODE_OUT_PATH/package.json"
-
-  pushd "$VSCODE_OUT_PATH"
-  symlink_asar
```

Comment:
should we also remove the function in lib.sh?

Reply:
I don't think so because we still need to symlink on the post-install AFAIK like @code-asher said. We only remove it here so the symlink isn't created when we build the release bundle.

Reply:
sounds good!